.travis.yml

```diff
@@ -5,28 +5,33 @@ os:
   - linux
   - osx
 
+git:
+  depth: 10
+  submodules: false
+
 go:
   - 1.8
 
 env:
-  - GOFLAGS=''
   - GOFLAGS='-race'
-  - GOFLAGS='--tags batched_queue'
   - GOFLAGS='-race --tags batched_queue'
-  - GOFLAGS=''                           WITH_ETCD=true
-  - GOFLAGS='-race'                      WITH_ETCD=true
-  - GOFLAGS='--tags pkcs11'              WITH_PKCS11=true
-  - GOFLAGS='-race --tags pkcs11'        WITH_PKCS11=true
+  - GOFLAGS='-race' WITH_ETCD=true
+  - GOFLAGS='-race --tags pkcs11' WITH_PKCS11=true
 
 matrix:
   fast_finish: true
   allow_failures:
     - os: osx
   exclude:
+    # Don't run optional build tags / features on OSX
```

Comment:
Good idea -- as long as we've got a basic OSX build (I develop on it sometimes so it's nice to have a build to catch any oddities), skipping the variants makes sense.

Reply:
Glad you liked it. :)

This and the xargs -P (for local builds) are the main reasons I took this PR forward.